specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/examples/Pricings/ListPricingsWithPlanFilter_example.json

```diff
@@ -0,0 +1,86 @@
+{
+  "parameters": {
+    "api-version": "2023-08-01-preview",
+    "scopeId": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "plansFilter": "VirtualMachines,KeyVaults"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/pricings/VirtualMachines",
+            "name": "VirtualMachines",
+            "type": "Microsoft.Security/pricings",
+            "properties": {
+              "pricingTier": "Standard",
+              "isEnabled": true,
+              "freeTrialRemainingTime": "PT0S",
+              "enablementTime": "2023-03-01T12:42:42.1921106Z",
+              "subPlan": "P2",
+              "enforce": false,
+              "extensions": [
+                {
+                  "name": "AgentlessVmScanning",
+                  "isEnabled": "True"
+                }
+              ]
+            }
+          },
+          {
+            "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/DEMO/providers/Microsoft.Compute/virtualMachines/VM-1/providers/Microsoft.Security/pricings/VirtualMachines",
+            "name": "VirtualMachines",
+            "type": "Microsoft.Security/pricings",
+            "properties": {
+              "pricingTier": "Free",
+              "isEnabled": false,
+              "freeTrialRemainingTime": "PT0S",
+              "enablementTime": "2023-03-01T12:42:42.1921106Z",
+              "subPlan": "P2",
```

Comment:
What is the meaning of sub-plan & extensions if the plan is disabled on this resource?